### ISSUE TITLE
[1.16.5] [16.2.0] Crash on Dedicated Servers

### ISSUE THREAD

**Issue by FoxMcloud5655** [stal111/Forbidden-Arcanus#201]
Client code is loading on dedicated servers, resulting in a crash:
```
Caught exception during event FMLCommonSetupEvent dispatch for modid forbidden_arcanus
java.lang.RuntimeException: Attempted to load class net/minecraft/client/world/ClientWorld for invalid dist DEDICATED_SERVER
	at net.minecraftforge.fml.loading.RuntimeDistCleaner.processClassWithFlags(RuntimeDistCleaner.java:71) ~[forge-1.16.5-36.2.8.jar:36.2]
	at cpw.mods.modlauncher.LaunchPluginHandler.offerClassNodeToPlugins(LaunchPluginHandler.java:85) ~[modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.ClassTransformer.transform(ClassTransformer.java:120) ~[modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader$DelegatedClassLoader.findClass(TransformingClassLoader.java:265) ~[modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:136) ~[modlauncher-8.0.9.jar:?]
	at cpw.mods.modlauncher.TransformingClassLoader.loadClass(TransformingClassLoader.java:98) ~[modlauncher-8.0.9.jar:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:351) ~[?:1.8.0_292]
	at java.lang.invoke.MethodHandleNatives.resolve(Native Method) ~[?:1.8.0_292]
	at java.lang.invoke.MemberName$Factory.resolve(MemberName.java:975) ~[?:1.8.0_292]
	at java.lang.invoke.MemberName$Factory.resolveOrFail(MemberName.java:1000) ~[?:1.8.0_292]
	at java.lang.invoke.MethodHandles$Lookup.resolveOrFail(MethodHandles.java:1394) ~[?:1.8.0_292]
	at java.lang.invoke.MethodHandles$Lookup.linkMethodHandleConstant(MethodHandles.java:1750) ~[?:1.8.0_292]
	at java.lang.invoke.MethodHandleNatives.linkMethodHandleConstant(MethodHandleNatives.java:477) ~[?:1.8.0_292]
	at com.stal111.forbidden_arcanus.network.NetworkHandler.init(NetworkHandler.java:27) ~[forbidden_arcanus:16.2.0]
	at com.stal111.forbidden_arcanus.ForbiddenArcanus.setup(ForbiddenArcanus.java:143) ~[forbidden_arcanus:16.2.0]
```

**Comment by ModernGamingWorld** [comment]
@stal111 Same crash for me On server side

**Comment by FoxMcloud5655** [comment]
My PR #202 fixes this issue.

**Comment by stal111** [comment]
Fixed in version 16.2.1
